components/prism-bash.js

```diff
@@ -113,6 +113,11 @@
 			alias: 'variable',
 			lookbehind: true
 		},
+		// Highlight parameter names as variables
+		'parameter': {
+			pattern: /[^|\s]-{1,2}(\w+:[+-]?)?\w+(?:\.\w+)*(?=[=\s]|$)/,
```

Comment:
You meant this, right?

(The `lookbehind: true` is necessary to make it so that the space before a parameter is not part of the parameter.)

```suggestion
			pattern: /(^|\s)-{1,2}(\w+:[+-]?)?\w+(?:\.\w+)*(?=[=\s]|$)/,
			lookbehind: true
```

Reply:
Yes.

Fixed.